tests/test_data/test_dataset.py

```diff
@@ -142,8 +149,34 @@ def setup_class(cls):
         ]
 
         cls.hvu_category_nums = [739, 117, 291, 69, 1679, 248]
+
+        cls.hvu_categories_for_eval = ['action', 'scene', 'object']
+        cls.hvu_category_nums_for_eval = [3, 3, 3]
+
         cls.filename_tmpl = 'img_{:05d}.jpg'
 
+    def test_rawvideo_dataset(self):
+        # Try to load txt file
+        rawvideo_dataset = RawVideoDataset(
+            ann_file=self.rawvideo_test_anno_txt,
+            pipeline=self.rawvideo_pipeline,
+            clipname_tmpl='part_{}.mp4',
+            sampling_strategy='positive',
+            data_prefix=self.data_prefix)
+        result = rawvideo_dataset.prepare_train_frames(0)
+        clipname = osp.join(self.data_prefix, 'test_rawvideo_dataset',
+                            'part_0.mp4')
+        assert result['filename'] == clipname
+
+        # Try to load json file
+        rawvideo_dataset = RawVideoDataset(
+            ann_file=self.rawvideo_test_anno_json,
+            pipeline=self.rawvideo_pipeline,
+            clipname_tmpl='part_{}.mp4',
+            sampling_strategy='random',
+            data_prefix=self.data_prefix)
+        result = rawvideo_dataset.prepare_test_frames(0)
```

Comment:
add `test_mode=True` and `result = rawvideo_dataset[0]`, unittest should be perform in the same way with the normal calling.